### DISCUSSION TITLE
lists of Public API which support  swagger 2.0 OR OpenAPI 3.0

### DISCUSSION THREAD

**Discussion by hegebharat** [schemathesis/schemathesis#1124] [General]
Hello ,

Do you have lists public apis for which support swager 2.0 OR Openapi 3.0 to test it. I have been looking into https://apis.guru/browse-apis/ but many apis need api-keys or billing account.

Please share us if you have such lists.

Thank you

**Comment by Stranger6667** [comment]
Hi!

At the moment, https://github.com/APIs-guru/openapi-directory is used for testing Schemathesis. It is the source repo for the link you provided. However, it is used in the `--dry-run` mode -> only data generation is tested.

In a few days, I plan to open source a project that will serve exactly this purpose - it contains open-source projects that could be built and tested by various web API fuzzers in a completely standalone fashion.

Regards

**Reply by hegebharat** [reply]
Thank you for the answer.
